js/ui/viewSelector.js

```diff
@@ -644,6 +647,18 @@ const ViewSelector = new Lang.Class({
         global.stage.add_action(gesture);
     },
 
+    _addViewsPageClone: function() {
+        let layoutViewsClone = new ViewsClone(this, this._viewsDisplay, false);
+        Main.layoutManager.setViewsClone(layoutViewsClone);
+
+        this._overviewViewsClone = new ViewsClone(this, this._viewsDisplay, true);
```

Comment:
I guess no? Again, not my code, so I just ported it and assumed this was needed for some reason. I'll try removing the reference though, can't find any other place this is needed, or a reason to keep a ref from here when the overview is already keeping one.